h2o-genmodel/src/main/java/hex/genmodel/attributes/parameters/IVariableImportancesHolder.java

```diff
@@ -0,0 +1,7 @@
+package hex.genmodel.attributes.parameters;
+
+import hex.genmodel.attributes.VariableImportances;
+
+public interface IVariableImportancesHolder {
```

Comment:
I dont think we follow the convention of prefixing interfaces with `I`

Reply:
Renamed to VariableImportancesHolder